src/context/App.tsx

```diff
@@ -173,7 +173,7 @@ export const AppProvider = ({ children }: { children: ReactNode }) => {
     const newSettings = { ...settings, ...defaultFilters };
     setSettings(newSettings);
     saveState({ auth, settings: newSettings });
-  }, [auth]);
+  }, [auth, settings]);
```

Comment:
Be careful with this 👀 All linters are known for sometimes inducing people in error with this, for example in useEffects

Reply:
Thanks. So for this instance do you think this should be reverted?